src/main/java/org/isf/medicalinventory/model/MedicalInventoryRow.java

```diff
@@ -156,6 +156,15 @@ public void setNewLot(boolean isNewLot) {
 	public void setRealQty(double realQty) {
 		this.realQty = realQty;
 	}
+	
+	public int getLock() {
+		return lock;
+	}
+
+	
+	public void setLock(int lock) {
+		this.lock = lock;
+	}
 
 	public String getSearchString() {
 		StringBuffer sbNameCode = new StringBuffer();
```

Comment:
Why in this method is `getMedical() != null` tested multiple times?  Why not test it once and if null return an empty string.